Description: Upstream changes introduced in version 2.07-6
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 fbi (2.07-6) unstable; urgency=low
 .
   * Switch to linux-any to ease the life for upcoming ports like
     Arm with hard float
   * Drop libpcd, which is to be removed.
 .
 The person named in the Author field signed this changelog entry.
Author: Moritz Muehlenhoff <jmm@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- fbi-2.07.orig/Make.config
+++ fbi-2.07/Make.config
@@ -5,7 +5,7 @@ HAVE_LINUX_FB_H	:= yes
 HAVE_NEW_EXIF	:= yes
 HAVE_GLIBC	:= yes
 HAVE_STRCASESTR	:= yes
-HAVE_LIBPCD	:= yes
+HAVE_LIBPCD	:= no
 HAVE_LIBUNGIF	:= yes
 HAVE_LIBPNG	:= yes
 HAVE_LIBTIFF	:= yes
